python/masonite/config/middleware.py

```diff
@@ -0,0 +1,33 @@
+"""Middleware Configuration Settings."""
+
+from masonite.middleware import ResponseMiddleware, MaintenanceModeMiddleware
+
+from app.http.middleware.AuthenticationMiddleware import \
+    AuthenticationMiddleware
+from app.http.middleware.CsrfMiddleware import CsrfMiddleware
+from app.http.middleware.LoadUserMiddleware import LoadUserMiddleware
+from app.http.middleware.VerifyEmailMiddleware import VerifyEmailMiddleware
+
+"""HTTP Middleware
+HTTP middleware is middleware that will be ran on every request. Middleware
+is only ran when a HTTP call is successful (a 200 response). This list
+should contain a simple aggregate of middleware classes.
+"""
+
+HTTP_MIDDLEWARE = [
+    # LoadUserMiddleware,
+    # CsrfMiddleware,
+    # ResponseMiddleware,
```

Comment:
If this benchmark is returning responses with the proper headers then the `ResponseMiddleware` needs to be uncommented

Reply:
I didn't know how did I missed it I will add it. Thanks for the review

Reply:
I added it back.